ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -56,12 +56,15 @@ class BadgeLabel: UILabel, TokenizedControlInternal {
     }
 
     private func updateColors() {
+        let colors = tokenSet.fluentTheme.aliasTokens.colors
         if shouldUseWindowColor {
```

Comment:
Possibly out of scope, but should there be a call to `updateColors` in the `didSet` of `shouldUseWindowColor`? Looks like if you set `shouldUseWindowColor` (which we could maybe name to `shouldUseThemeColor` or `shouldUseBrandColor`) after the theme has been set or after it has moved to a window, it wouldn't get updated.